aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -586,12 +584,52 @@ module std::features {
         is_enabled(ABORT_IF_MULTISIG_PAYLOAD_MISMATCH)
     }
 
-    /// Whether the simulation enhancement is enabled. This enables the simulation without an authentication check,
-    /// the sponsored transaction simulation when the fee payer is set to 0x0, and the multisig transaction
-    /// simulation consistnet with the execution.
+    const TRANSACTION_SIMULATION_ENHANCEMENT: u64 = 78;
+    /// Whether the Atomic bridge is available
+    /// Lifetime: transient
+    const ATOMIC_BRIDGE: u64 = 224;
+
+    #[deprecated]
+    public fun get_atomic_bridge_feature(): u64 { ATOMIC_BRIDGE }
+
+    #[deprecated]
+    public fun abort_atomic_bridge_enabled(): bool {
+        true
+    }
+
+
+    /// Whether the Atomic bridge is available
+    /// Lifetime: transient
+    const NATIVE_BRIDGE: u64 = 225;
+
+    #[deprecated]
+    public fun get_native_bridge_feature(): u64 { NATIVE_BRIDGE }
+
+    #[deprecated]
+    public fun abort_native_bridge_enabled(): bool {
+        true
+    }
+
+    /// Whether the Governed Gas Pool is used to capture gas fees
     ///
+    /// Lifetime: permanent
+    const GOVERNED_GAS_POOL: u64 = 223;
```

Comment:
The GGP feature value is 73 on `movement-migration` branch. Is it ok to change it? There's no risk with the db state?
I saw that you disable the old value and enable the new one. The feature value isn't anywhere else?